README.md

```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
 ```
 
 ---
+### How it works
```

Comment:
I think this whole section should probably be before "Benchmarks".  Also, please match the spacing between the H3 and the divider line as shown in the other sections:

```markdown
... lorem ipsum dolor sit amet.

---

### Some title
Lorem ipsum dolor sit amet...
```